relay_rpc/src/auth/cacao/header.rs

```diff
@@ -1,20 +1,21 @@
 use {
     super::CacaoError,
     serde::{Deserialize, Serialize},
+    std::sync::Arc,
 };
 
 pub const EIP4361: &str = "eip4361";
 
 #[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize, Hash)]
 pub struct Header {
-    pub t: String,
+    pub t: Arc<str>,
```

Comment:
Upgrading to `Arc` to avoid re-allocating to serve the error. Easy win, though minor